tests/std/tests/P0553R4_bit_rotating_and_counting_functions/test.cpp

```diff
@@ -105,6 +105,27 @@ constexpr bool test_rotr() {
     return true;
 }
 

```

Comment:
Can we unskip the P0553R4 tests in `tests\libcxx\expected_results.txt` and `skipped_tests.txt`?